e2e-tests/tfm/cypress/e2e/journeys/case-amendments/facility-end-date/feature-flag-disabled/gef-deal-expect-no-facility-end-date-features.spec.js

```diff
@@ -0,0 +1,85 @@
+import relative from '../../../../relativeURL';
+import facilityPage from '../../../../pages/facilityPage';
+import { ADMIN, BANK1_MAKER1, PIM_USER_1, T1_USER_1 } from '../../../../../../../e2e-fixtures';
+import { MOCK_APPLICATION_AIN } from '../../../../../fixtures/mock-gef-deals';
+import { MOCK_FACILITY_ONE } from '../../../../../fixtures/mock-gef-facilities';
+import { DEAL_TYPE } from '../../../../../../../gef/cypress/fixtures/constants';
+import amendmentsPage from '../../../../pages/amendments/amendmentsPage';
+import dateConstants from '../../../../../../../e2e-fixtures/dateConstants';
+
+if (Cypress.env('FF_TFM_FACILITY_END_DATE_ENABLED') === 'false') {
+  context('Amendments - GEF deal does not display any Facility end date pages or fields when TFM feature flag disabled', () => {
+    let dealId;
+    let dealFacilities;
+
+    const MOCK_GEF_FACILITY_WITH_FACILITY_END_DATE = {
+      ...MOCK_FACILITY_ONE,
```

Comment:
Do we want to directly update these mocks (conditionally on the env var)?

Reply:
Ooh interesting, could you explain more what you mean here - in this file this will only run if the env file has the value as `false` so I don't think it would do much in this instance, unless you have a different sort of refactor in mind?

Reply:
Thinking a la https://github.com/UK-Export-Finance/dtfs2/pull/3406/files#r1729157033 conditionally add these values directly to the mocks

Reply:
added